wire-ios-data-model/Source/Proteus/ProteusProvider.swift

```diff
@@ -70,10 +74,14 @@ public class ProteusProvider: ProteusProviding {
 
           return try proteusServiceBlock(proteusService)
 
-        } else {
+        } else if !proteusViaCoreCrypto {
 
             // remove comment once implementation of proteus via core crypto is done
             return try keyStoreBlock(keyStore)
+        } else {
+
+            WireLogger.coreCrypto.error("can't access any proteus cryptography service")
+            fatal("can't access any proteus cryptography service")
```

Comment:
Do we want the app to crash?

Reply:
because the method requires a T it's the only way to make it work. Also I would expect this to be a developer error to fall into this case